client/app/scripts/charts/edge.js

```diff
@@ -6,7 +6,7 @@ import { enterEdge, leaveEdge } from '../actions/app-actions';
 import { encodeIdAttribute, decodeIdAttribute } from '../utils/dom-utils';
 
 function isStorageComponent(id) {
-  if (id === '<persistent_volume>' || id === '<storage_class>' || id === '<persistent_volume_claim>') {
+  if (id === '<persistent_volume>' || id === '<storage_class>' || id === '<persistent_volume_claim>' || id === '<storage_pool>' || id === '<disk>' || id === '<storage_pool_claim>') {
```

Comment:
It looks messy now!  Use `includes` method in js to standardized this implementation.

```js

function isStorageComponent(id) {
  var storageComponents =  ['<persistent_volume>', '<storage_class>', '<persistent_volume_claim>', '<storage_pool>', '<disk>', '<storage_pool_claim>']
    return storageComponents.includes(id)
}
```